proto/grp.proto

```diff
@@ -17,4 +20,45 @@ message Group {
 
   // True if this group has linked a Github token.
   bool github_linked = 4;
+
+  // Whether users from the owned domain should automatically
+  // be added to the group when they sign up (defaults to false).
+  //
+  // Not yet implemented.
+  bool auto_populate_from_owned_domain = 5;
+}
+
+message CreateGroupRequest {
+  // Group initialized with required fields except for ID.
+  //
+  // The required fields are documented in the Group proto.
+  Group group = 1;
+}
+
+message CreateGroupResponse {
+  // Whether group creation was successful.
+  boolean success = 1;
+
+  // The ID of the created group.
+  string group_id = 2;
+
+  // Error messages associated with problematic form fields (if creation wasn't
```

Comment:
I like the idea behind this but I think it's overkill since we have so few fields. To keep things simple, can we remove the errors field and use the overall RPC status to handle errors? The client can display the status message to the user if the status code matches InvalidRequest.

Reply:
SG